cmd/fdsn-ws-nrt/data_nrt_test.go

```diff
@@ -194,7 +194,7 @@ func BenchmarkGetRecordDB(b *testing.B) {
 }
 
 // funcs for setting up test data.
-
+//nolint:deadcode
```

Comment:
Please add reasons to all your exclusions. If you put a second comment it won't affect the linter. i.e:
```
//nolint:deadcode //[reason]
```